src/desktop/apps/artwork/client.tsx

```diff
@@ -87,7 +90,7 @@ mediator.on("launchInquiryFlow", options => {
 
 mediator.on("openBuyNowAskSpecialistModal", options => {
   // TODO: Remove after inquiry a/b test
-  splitTest("inquiry_auth").view()
+  maybeFireTestView()
```

Comment:
Why are we firing an experiment view in the buy now flow? Has this flow changed?